src/plot.cpp

```diff
@@ -1,10 +1,1010 @@
+#include <fstream>
+
 #include "openmc/plot.h"
+#include "openmc/constants.h"
+#include "openmc/settings.h"
+#include "openmc/error.h"
+#include "openmc/particle.h"
+#include "openmc/geometry.h"
+#include "openmc/cell.h"
+#include "openmc/material.h"
+#include "openmc/string_functions.h"
+#include "openmc/mesh.h"
+#include "openmc/output.h"
+#include "openmc/hdf5_interface.h"
+#include "openmc/random_lcg.h"
+#include "openmc/output.h"
 
 namespace openmc {
 
+//===============================================================================
+// Global variables
+//===============================================================================
+
+int PLOT_LEVEL_LOWEST = -1;
+
+std::map<int, int> plot_map;
+
+int n_plots;
```

Comment:
I've tried it and it is, but its still part of the CAPI right now. I figured there would be a cleanup of that later and wanted to continue to support it until then. Should I go ahead and remove it?

Reply:
Oh, didn't realize it was a C-API variable.  We can leave it in for now, but make sure to declare it as `extern "C" int n_plots` so that it is actually defines the C-API variable rather than a new `n_plots` variable.